vizro-core/docs/pages/user-guides/assets.md

```diff
@@ -160,6 +163,9 @@ Suppose you want to hide the page title on one page only. Here's how you can ach
         dashboard = vm.Dashboard(pages=[page_one, page_two])
         Vizro().build(dashboard).run()
         ```
+
+        <img src=https://py.cafe/logo.png alt="py.cafe logo" width="30"><b><a target="_blank" href="https://py.cafe/stichbury/vizro-multi-page-example">Run and edit this code in Py.Cafe</a></b>
```

Comment:
Yes agreed, would not link to any of my examples at the very least. At best all in our new profile

Reply:
OK, this is fine but it would have been better if I hadn't spent a day on this PR coding the projects and links only to now recreate it all. Sorry to be grumpy but I don't have time to switch them all around again. Can we not leave it until we need to revise them and next time they get touched put them in a new profile. I am on PTO soon and there's too much else stacking up before I go which I've not got to because of the PR 😓

Reply:
That also sounds good to me! No need to change anything now! Let's say, all in your account, and we do a quick transfer once we next touch them?